### ISSUE TITLE
Info endpoint contribution enhancements

### ISSUE THREAD

**Issue by metacosm** [quarkusio/quarkus#43788]
### Description

- Validate / issue warning when multiple contributions declare the same name (as, right now, they get replaced by the newest one, resulting in random results, depending on the processing order)
- Separate name and display name that gets displayed in the dev UI cards
- Make it possible to specify an icon, ideally automatically picked up from the resources directory using a simple naming convention (e.g. info-contributor-<name>-icon.svg)

cc @phillip-kruger

**Comment by phillip-kruger** [comment]
I can look at this, but I think the info extension is done by @cescoffier and/or @geoand . I`ll let them comment first

**Comment by metacosm** [comment]
Yeah, also started looking into moving `InfoEndpointEnabled` into the SPI module but this doesn't seem as straightforward as just moving the class.

**Comment by geoand** [comment]
I can look into this at some point, but don't expect it to be soon

**Comment by metacosm** [comment]
I might be able to take this on, but might need some guidance.

**Comment by geoand** [comment]
https://github.com/quarkusio/quarkus/pull/43868 seems fine to me for the duplicate values
